docs/guides/latest-updates.mdx

```diff
@@ -124,17 +124,20 @@ Users can purchase licenses for the following functions contributed by our partn
 To get started, explore the [Qiskit Functions documentation](/guides/functions).
 
 
-## IBM Quantum blog: IBM Quantum delivers on performance challenge
+## IBM Quantum blog: Two new blogs in January
 
 *Browse all blogs at the [IBM Quantum blog page](https://www.ibm.com/quantum/blog).*
 
 {/* Bring over the Quantum news (blog) announcement post */}
 
-Two years ago, IBM set a challenge for both itself and the quantum community. We asked quantum researchers to work on developing quantum algorithms incorporating circuits with 100 qubits and gate depths of 100. In exchange, we would build a quantum computer capable of returning accurate values for those circuits in less than a day’s runtime.
+The first IBM Quantum blog post of the year takes a look back at some of the amazing tools released in 2024 that help make quantum software development easier and more accessible than ever before.
+The blog provides recaps of major releases from last year, including Qiskit v1.x, Qiskit addons, Qiskit Transpiler Service, and more - all with code examples and documentation links to help readers get started using them.
 
-On 11 November 2024, at the first-ever IBM Quantum Developer Conference (QDC), IBM announced the fulfillment of that promise: a quantum computer capable of running quantum circuits with up to 5,000 two-qubit gate operations.
+*Blog: [A year of simplifying quantum software development tools with Qiskit](https://www.ibm.com/quantum/blog/dev-tools)*
 
-On the IBM Quantum blog, we take a deep dive into the many hardware advances, software innovations, and research partnerships that have made this achievement possible, and offer a preview of what's coming next. Visit the [blog](https://www.ibm.com/quantum/blog/qdc-2024) to learn more.
+Next, read how dynamic circuits can enable remarkably efficient long-range entanglement between qubits in a quantum chip. The method was first demonstrated by IBM researchers via a recent paper published in PRX Quantum, and the researchers behind it say their technique could greatly enhance the level of complexity we can represent in quantum circuits through qubit entanglement. The blog explores the power of dynamic circuits and explains how the new method differs from traditional approaches for establishing entanglement between qubits that are not directly connected to each other.
```

Comment:
Would the second sentence be more clear as two distinct sentences?

Reply:
Possibly - the comma does some heavy lifting - but since the author's words are already published in this form in the dashboard announcements, I'll leave it as-is :)